src/database/hooks/useLocalDatabaseHook.js

```diff
@@ -10,14 +10,15 @@ import {InitialStartupAtom} from '../../service/initialStartup';
 
 const MIGRATION_STATUS = {
   NOT_MIGRATED: 'NOT_MIGRATED',
-  MIGRATED: 'MIGRATED'
+  MIGRATED: 'MIGRATED',
+  MIGRATING: 'MIGRATING'
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a new migration status 'MIGRATING' is a good practice. It allows the system to handle cases where the database is in the process of migrating, preventing any operations that could potentially interfere with the migration process.

```diff
+  MIGRATING: 'MIGRATING'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->